release_notes/ocp-4-6-release-notes.adoc

```diff
@@ -196,6 +196,27 @@ By default, the following network connections are checked:
 ** the internal API load balancer
 ** the external API load balancer
 
+[id="ocp-4-6-storage"]
+=== Storage
+
+[id="ocp-4-6-csi-driver-cso"]
+==== CSI drivers now managed by the Cluster Storage Operator
+The Container Storage Interface (CSI) drivers for AWS Elastic Block Store (EBS), OpenStack Manila shared file system service, and Red Hat Virtualization (oVirt) are now managed by the Cluster Storage Operator in the {product-title} back end.
+
+This feature installs the CSI Driver Operator and the CSI driver in the `openshift-cluster-csi-drivers` namespace by default, allowing you to create CSI-provisioned volumes that mount to assets in use by AWS EBS, Manila, and oVirt.
+
+[IMPORTANT]
+====
+If you installed any CSI Operators and CSI drivers on an {product-title} 4.5 cluster, you must uninstall them before you update to a newer version of {product-title}.
```

Comment:
For manila csi driver is a GA feature in 4.5, so we don't need to uninstall manila csi operator and driver before upgrade from 4.5 to 4.6

Reply:
Ah, right...thanks. I've updated to remove Manila from this note.